pmoves/tools/secrets_funnel_populate.py

```
@@ -0,0 +1,845 @@
+]
+
+# Sentinel values that indicate "not set"
+SENTINEL_VALUES = frozenset({
+    "unset-pending-key",
+    "local-disabled",
+    "placeholder",
+    "",
+    "null",
+    "none",
+    "todo",
+    "",
```

Ruff (0.15.20)

[warning] Sets should not contain duplicate item `""`

Remove duplicate item

(B033)